package.json

```diff
@@ -93,6 +93,16 @@
         "category": "Metals",
         "title": "Rescan sources"
       },
+      {
+        "command": "metals.compile-cascade",
+        "category": "Metals",
+        "title": "Cascade compile"
+      },
+      {
+        "command": "metals.compile-cancel",
+        "category": "Metals",
+        "title": "Cancel compilation"
+      },
```

Comment:
We should also add these to the `menus.commandPalette` section of the manifest, so that they only appear when the selected buffer is a Scala file